### DISCUSSION TITLE
Migration wallet problems

### DISCUSSION THREAD

**Discussion by Tonnie80** [iotaledger/legacy-migration-tool#133] [Help & Support]
I have a problem! I still have coins from the IOTA ligt wallet 2.2.6, but when I enter the 81 seed into the migration wallet, it shows a balance of 0. Is there a solution?  I can't see the balance anymore because the old IOTA address has 90 characters

**Comment by begonaalvarezd** [comment]
You can look for your legacy balances in the explorer legacy archive https://explorer.iota.org/legacy-mainnet/
You might have migrated already, check if your addresses have tx history at around or after april 2021, if so, you migrated already

**Reply by Tonnie80** [reply]
Thanks for your comment. The balance is 0. This is my address: AZSJPRDNWOD9BHNKONUPNQYFDCVDYKUVSYJGBZUYHEMA9TITXVGXBQYPQGMNJKFNEBOOMQGBPUS9OZYQ9ZSPMNBHCZ
What if my address has no Tx history? have I lost my coins now?

**Reply by begonaalvarezd** [reply]
You shouldnt have lost anything, are you 100% that is your latest address with funds?

**Reply by Tonnie80** [reply]
Yes, it worked in the old Explorer, I have had the coins since 2017

**Reply by begonaalvarezd** [reply]
I suggest you join our discord for further support https://discord.iota.org/, you can open a ticket in the `#help` channel

**Reply by Tonnie80** [reply]
Okay thanks
